fusor-ember-cli/app/templates/review/progress/overview.hbs

```diff
@@ -1,26 +1,15 @@
 <br />
 
-{{progress-bar name=nameSatellite
-               isSatelliteProgressBar=true
-               model=model.manageContentTask
-               synctask0=model.synctask0
-               synctask1=model.synctask1
-               synctask2=model.synctask2
-               synctask3=model.synctask3
```

Comment:
These subtasks aren't used anyway. I was calculating the average of the "Manage Content" tasks but not doing that anymore.